.circleci/config.yml

```diff
@@ -0,0 +1,101 @@
+version: 2
+
+.base: &base
+  docker:
+      - image: trojsten/web-base:latest
+      - image: postgres:9.5
+        environment:
+          POSTGRES_USER: trojsten
+          POSTGRES_DB: trojsten
+      # - image: elasticsearch:2.4.6
+  working_directory: ~/web
+
+
+jobs:
+  checkout_code:
+    <<: *base
+    steps:
+      - checkout
+      - save_cache:
+          key: v1-repo-{{ .Environment.CIRCLE_SHA1 }}
+          paths:
+            - ~/web
+  build_dependencies:
```

Comment:
can't we merge the checkout_code and build_dependencies steps? it seems unnecessary as the checkout_code doesn't actually test anything worth testing and I think it prolongs the CI process by around half a minute

Reply:
Done.